other/materials_designer/development_notebook.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>Let's call this <code>custom_transformation</code> or <code>arbitrary_transformation</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>